frontend/src/components/questions/UploadQuestion.vue

```diff
@@ -56,7 +56,7 @@ function onDragLeave(event: DragEvent) {
 
 function onFileAdd(event: Event) {
     event.preventDefault();
-    const fileList = getFileList(<HTMLInputEvent | DragEvent>event);
+    const fileList = getFileList(event as HTMLInputEvent | DragEvent);
```

Comment:
I talked too quick [here](https://github.com/YacineSteeve/quizoot/pull/50#discussion_r1046436353) 😂 way much clear to me.